Update security.md (#152)
MicrosoftDocs/well-architected: well-architected/azure-vmware/security.md

# Security considerations for Azure VMware Solution workloads

This article discusses the security design area of an Azure VMware Solution workload. The discussion covers various measures for helping to secure and protect Azure VMware Solution workloads. These measures help protect infrastructure, data, and applications. This approach to security is holistic and aligns with an organization's core priorities.

Securing Azure VMware Solution requires a shared responsibility model, where Microsoft Azure and VMware are responsible for certain security aspects. To implement appropriate security measures, ensure a clear understanding of the shared responsibility model and collaboration between IT teams, VMware, and Microsoft.

## Manage compliance and governance

*Impact: Security, Operational Excellence*

> To avoid deleting the private cloud by mistake, use [resource locks](https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/lock-resources?tabs=json) to safeguard resources from unwanted deletion or change. They can be set at the subscription, resource group, or resource level and block deletions, modifications, or both. 

Reviewer: ```suggestion
To avoid deleting the private cloud by mistake, use [resource locks](/azure/azure-resource-manager/management/lock-resources?tabs=json) to safeguard resources from unwanted deletion or change. They can be set at the subscription, resource group, or resource level and block deletions, modifications, or both. 
```

> It's also important to detect non-compliant servers. You can use Azure Arc for this purpose. Azure Arc extends Azure management capabilities and services to on-premises or multi-cloud environments. Azure Arc gives you a single-pane view for applying updates and hotfixes by providing centralized server management and governance. The result is a consistent experience for managing components from Azure, on-premises systems, and Azure VMware Solution.

Reviewer 2: ```suggestion
It's also important to detect noncompliant servers. You can use Azure Arc for this purpose. Azure Arc extends Azure management capabilities and services to on-premises or multicloud environments. Azure Arc gives you a single-pane view for applying updates and hotfixes by providing centralized server management and governance. The result is a consistent experience for managing components from Azure, on-premises systems, and Azure VMware Solution.
```
Reverting a few terms to align with Cloud + AI style guide.

*Impact: Security*

> If you don't patch and regularly update your operating system, you make it susceptible to vulnerabilities and put your entire platform at risk. When you apply patches regularly, you keep your system up to date. When using an endpoint protection solution, you help prevent common attack vectors from targeting your operating system. It's also important to regularly perform vulnerability scans and assessments. These tools help you identify and remediate security weaknesses and vulnerabilities.

Reviewer 2: ```suggestion
If you don't patch and regularly update your operating system, you make it susceptible to vulnerabilities and put your entire platform at risk. When you apply patches regularly, you keep your system up to date. When you use an endpoint protection solution, you help prevent common attack vectors from targeting your operating system. It's also important to regularly perform vulnerability scans and assessments. These tools help you identify and remediate security weaknesses and vulnerabilities.
```
Fixing grammar to avoid misplaced modifier.

##### Recommendations

- Use network segments to separate and monitor components logically.

> - Use micro-segmentation capabilities native to VMware NSX-T Data Center to restrict network communication between application components.

Reviewer 2: ```suggestion
- Use micro-segmentation capabilities that are native to VMware NSX-T Data Center to restrict network communication between application components.
```
Reverting the change to preserve Microsoft style for Global English.
